README.md

```diff
@@ -1,13 +1,10 @@
 # Project Labyrinth
+HTML, CSS, JS, REACT, REDUX
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Fetched Api address.
+Used Thunks and Dispatch with Redux. 
+Used Lottie animation.
+Useselector and Globalstyles used as well.
 
 ## View it live
 
```

Comment:
Missing netlify link to view it somewhere, do you have it? :) it is a little bit difficult to test the game out without one unfortunately